service-deployment/bootstrap/hadoop-service/hadoop-configuration/jobhistory-generate-script.sh

```diff
@@ -20,6 +20,20 @@
 cp  /hadoop-configuration/jobhistory-mapred-site.xml $HADOOP_CONF_DIR/mapred-site.xml
 cp  /hadoop-configuration/jobhistory-yarn-site.xml $HADOOP_CONF_DIR/yarn-site.xml
 
+# Step 4(3) of 4 to set up Hadoop queues.
+# Patch the original capacity-scheduler.xml file with the one in the configmap.
+# Three sub-steps:
+#   1) Extract the head part from the original capacity-scheduler.xml file (a.k.a. the target file).
+#   2) Extract the 'configuration' part from the capacity-scheduler.xml in the configmap (a.k.a. the content file).
+#   3) Concatent the above two parts and use it to replace the original capacity-scheduler.xml file.
+# Do the same for resourcemanager and nodemanager.
```

Comment:
Looks this part is common to 3 services, can you put them into a common script and call from 3 services to avoid code duplication?